src/bin/ayb.rs

```diff
@@ -129,6 +130,27 @@ async fn main() -> std::io::Result<()> {
                                 .default_value(OutputFormat::Table.to_str())
                                 .required(false)),
                 )
+                .subcommand(
+                    Command::new("profile")
+                        .about("Show the profile of an entity")
+                        .arg(arg!(<entity> "The entity to query")
+                            .required(true))
+                        .arg(
+                            arg!(--format <type> "The format in which to output the result")
+                                .value_parser(value_parser!(OutputFormat))
+                                .default_value(OutputFormat::Table.to_str())
+                                .required(false))
+                )
+                .subcommand(
+                    Command::new("update_profile")
+                        .about("Update the profile of an entity")
+                        .arg(arg!(<entity> "The entity to update").required(true))
+                        .arg(arg!(--display_name <value> "New display name").required(false))
+                        .arg(arg!(--description <value> "New description").required(false))
+                        .arg(arg!(--organization <value> "New organization").required(false))
+                        .arg(arg!(--location <value> "New location").required(false))
+                        .arg(arg!(--link <value> "New URL, will overwrite previous ones").required(false))
```

Comment:
given you can have multiple links, should it be possible to send a list with `.num_args(0..)`? we could call it `--links` at that point as well.

Reply:
It seems like we sort of do that below with `get_many`, so this might instead be feedback on the documentation string, which made me think we can only accept one link, and it will overwrite all other links.